assets/src/edit-story/components/panels/design/preset/colorPreset/colorPresetActions.js

```diff
@@ -214,10 +220,10 @@ function ColorPresetActions({ color, pushUpdate }) {
           <CtaWrapper>
             <TranslateWithMarkup
               mapping={{
-                i: <Add />,
+                b: <Strong />,
               }}
             >
-              {__('Click <i></i> to save a color', 'web-stories')}
+              {__('Click <b>+</b> to save a color', 'web-stories')}
```

Comment:
@miina, while looking to fix this, I noticed that the actual **+** used in the design is a real text plus just in a bigger font and in white. So I did the same here and it looks pretty similar:

| Design | Implementation |
|-|-|
| ![Screen Shot 2021-02-11 at 20 49 07](https://user-images.githubusercontent.com/637548/107721439-a2699280-6caa-11eb-9f69-444c5fb132ea.png) | ![Screen Shot 2021-02-11 at 20 48 57](https://user-images.githubusercontent.com/637548/107721441-a3022900-6caa-11eb-83a1-a2a45667d4d3.png) |

Reply:
Sounds good to me, the approach. Looks like it needs a bit more room though and looks thicker than on the design. Maybe reduce the font size to 20 and/or make it inline-block to assign fixed width? (I can look at it within #6208 as well though, not critical!)

Reply:
Let's deploy as-is for 1.4 and we can fix in #6208 when redoing the component anyway.